src/particles/PlasmaParticleContainer.cpp

```diff
@@ -116,11 +116,26 @@ PlasmaParticleContainer::ReadParameters ()
             m_u_mean[idim] = loc_array[idim];
         }
     }
-    if (queryWithParser(pp, "u_std", loc_array)) {
+    bool thermal_momentum_is_specified = queryWithParser(pp, "u_std", loc_array);
+    bool temperature_is_specified = queryWithParser(pp, "temperature_in_ev", m_temperature_in_ev);
+    if (thermal_momentum_is_specified) {
+        AMREX_ALWAYS_ASSERT_WITH_MESSAGE( temperature_is_specified == 0,
+        "Please specify exlusively either a temperature or the thermal momentum");
         for (int idim=0; idim < AMREX_SPACEDIM; ++idim) {
             m_u_std[idim] = loc_array[idim];
         }
     }
+
+    if (temperature_is_specified) {
+        AMREX_ALWAYS_ASSERT_WITH_MESSAGE( thermal_momentum_is_specified == 0,
+            "Please specify exlusively either a temperature or the thermal momentum");
+        const PhysConst phys_const_SI = make_constants_SI();
+        for (int idim=0; idim < AMREX_SPACEDIM; ++idim) {
+            m_u_std[idim] = sqrt( (m_temperature_in_ev * phys_const_SI.q_e)
+                                /(phys_const_SI.m_e * phys_const_SI.c * phys_const_SI.c ) );
```

Comment:
This doesn't have factor of 3/2. As I understand it, the factor of 1/2 isn't there because we are looking at momentum, and the factor of 3 just comes from the fact that we have 3 directions, so I believe this is correct, just double-checking.